apps/web/src/utils/validators.ts

```diff
@@ -0,0 +1,31 @@
+import type { NumberRange } from "~/components/RangeInput";
+
+export const validateRange = (
```

Comment:
are we using this func? I think this was part of previous implementation. Should we delete it?

Reply:
Oops, this is a leftover file from the main branch merge.